dev/integration_tests/android_driver_test/test_driver/main_test.dart

```diff
@@ -16,6 +18,8 @@ void main() async {
   await testExecutable(_main);
 }
 
+final bool _isLuciCi = io.Platform.environment['LUCI_CI'] == 'true';
```

Comment:
LUCI sets `LUCI_CI` to `True` with a capital T

(see a dump of the LUCI environment variables from this recipe at https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8740683707278457649/+/u/run_test.dart_for_flutter_driver_android_shard_and_subshard_None/execution_details)

Reply:
Sigh, thank you.